mgmtd/mgmt_vty.c

```diff
@@ -470,6 +533,7 @@ void mgmt_vty_init(void)
 	install_element(CONFIG_NODE, &mgmt_delete_config_data_cmd);
 	install_element(CONFIG_NODE, &mgmt_load_config_cmd);
 	install_element(CONFIG_NODE, &mgmt_save_config_cmd);
+	install_element(CONFIG_NODE, &mgmt_copy_config_cmd);
```

Comment:
In the doc comment you say this is both exec and config command. Here's it's added to config node only.

Reply:
I divided this into two separate command now. 'mgmt copy-config running startup' under exec mode (just like write men is) and 'mgmt auto-copy-config running startup' in config mode to enable auto-saving of startup file.